src/main.rs

```diff
@@ -33,6 +33,14 @@ fn main() {
                 .value_name("CMD")
                 .help("run a dijo command"),
         )
+        .arg(
```

Comment:
I would like to expand this to a full blown query system, using format strings.

Reply:
Do you need that for this PR or is it an idea for the future?

Reply:
This PR is good as it is, the query system and more command line flags can come later.